components/expenses/PayExpenseModal.tsx

```diff
@@ -150,17 +153,23 @@ const getPayoutLabel = (intl, type) => {
   return i18nPayoutMethodType(intl, type, { aliasBankAccountToTransferWise: true });
 };
 
+const DEFAULT_PAYMENT_METHOD_SERVICE = {
+  [PayoutMethodType.PAYPAL]: 'PAYPAL',
+  [PayoutMethodType.BANK_ACCOUNT]: 'WISE',
+};
+
 const getPayoutOptionValue = (payoutMethod, isAuto, host) => {
   const payoutMethodType = payoutMethod?.type;
   if (payoutMethodType === PayoutMethodType.OTHER) {
-    return { forceManual: true, action: 'PAY' };
+    return { forceManual: true, action: 'PAY', paymentMethodService: null };
   } else if (payoutMethod?.data?.type === 'brazil') {
     // TODO: remove this when we implement the missing Brazilian TRANSFER NATURE field.
     return { forceManual: true, action: 'PAY' };
   } else if (payoutMethodType === PayoutMethodType.BANK_ACCOUNT && !host.transferwise) {
-    return { forceManual: true, action: 'PAY' };
+    return { forceManual: true, action: 'PAY', paymentMethodService: 'WISE' };
```

Comment:
For consistency we should keep service + type, we tried to get rid of this in the past but any departure from that created confusion.

In those cases, maybe this should be `wise` + `manual`, and `paypal` + `manual` (to potentially differenciate from` adaptive` and `payout`)

Reply:
Since we only require this information on manual payments and the type will always be manual, the type is being set at the backend. In practice, `paymentMethodService` will only be considered by the backend when `forceManual` is also true.